genai-perf/tests/test_config_command.py

```diff
@@ -0,0 +1,272 @@
+# Copyright 2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from unittest.mock import patch
+
+# Skip type checking to avoid mypy error
+# Issue: https://github.com/python/mypy/issues/10632
+import yaml  # type: ignore
+from genai_perf.config.input.config_command import ConfigCommand, Range
+from genai_perf.inputs.input_constants import ModelSelectionStrategy, OutputFormat
+
+
+class TestConfigCommand(unittest.TestCase):
```

Comment:
I have a concern about using unittest here. Pytest and unittest are compatible but the structure of the testing has different patterns now.
Was unittest used here for a specific functionality that is harder in pytest? OR is it what you are more familiar with?

Reply:
I just don't know pytest and unittest is fully compatible in a pytest environment. I'm not doing anything fancy here, so there's no reason to use unit test other than it's what I'm familiar with.

FYI, all the code w/ analyze I added is done in unittest so unfortunately we already have a mix of testing in GAP.

Reply:
okay. perhaps not ideal but its the most extensive testing we have in place so im not gonna complain about it. Thanks for chatting thru it with me.